### ISSUE TITLE
Pets don't regen the health that is described in the stat window...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#6694]
> Pets don't regen the health that is described in the stat window when you hover over "Spirit". Overall they practically don't regen requiring spamming of mend pet.

.go xyz 6487.140000 -3193.360000 569.245000 1 3.424240

**Comment by zana244** [comment]
Confirmed on internal, a bit unsure as to what the proper values should be though... A lot of people complained of slow pet regen.
At lvl 57 my wolf pet has 76 spirit which says it should regen 44 HP per second, but on Epoch it regens 19 every 3 sec...
On stormforge, at lvl 70 my pet regens 55 HP per second on tooltip, and I got 158 HP every 3 sec.
On VMangos, my 60 Owl regens 312 hp every 5 sec, which is about 62-63 hp per sec.

Possibly the same for warlocks, however when I tried a Voidwalker at lvl 30 the values in the stat window were low from the beginning, 5 HP per second, and the VW was regenerating around 12 hp every 3 sec.
